File: app/src/main/java/com/stevenfrew/beatprompter/midi/StopMessage.java
package com.stevenfrew.beatprompter.midi;

class StopMessage extends OutgoingMessage
{
    private final static byte[] MIDI_STOP_SIGNAL_BYTES=new byte[]{0x0f, Message.MIDI_STOP_BYTE,0,0};
    StopMessage()
    {
        super(MIDI_STOP_SIGNAL_BYTES,true);
    }
}

File: app/src/main/kotlin/com/stevenfrew/beatprompter/midi/StopMessage.kt
package com.stevenfrew.beatprompter.midi

internal class StopMessage : OutgoingMessage(MIDI_STOP_SIGNAL_BYTES, true) {
    companion object {
        private val MIDI_STOP_SIGNAL_BYTES = byteArrayOf(0x0f, Message.MIDI_STOP_BYTE, 0, 0)
    }
}